x/crosschain/types/keys.go

```diff
@@ -25,6 +25,8 @@ const (
 	MemStoreKey = "mem_metacore"
 
 	ProtocolFee = 2000000000000000000
+
+	TssMigrationGasMultiplierEVM = "2.5"
```

Comment:
Can we add comment

Reply:
1a24f7336d587480ed91c3bb65541a2fdecc01ae